scene/scene_string_names.h

```diff
@@ -125,6 +125,7 @@ class SceneStringNames {
 	StringName _ready;
 	StringName _unhandled_input;
 	StringName _unhandled_key_input;
+	StringName _propagate_input_event;
```

Comment:
This isn't needed for anything.

Reply:
I'm not sure, I can follow your argument. The StringName is used within the `GDVIRTUAL_CALL`.
When I looked at other `GDVIRTUAL_CALL` functions, they all use StringName for their first argument!?

Reply:
They don't, check the macro expansion. Seems like the other stringnames are leftover from the old virtual method calls and are unused now.

Reply:
Wow, thanks, It works even without that StringName definition. I wasn't aware that this is from legacy code.